ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -346,6 +346,15 @@ classes:
                 type: lines
                 index_type: keyword
 
+            storage_disk_lun:
+                # link to storage disk/virtual drive
+                # if the hard disk is actually mapped to it
+                label: Storage Disk/LUN
+                type: entity
+                grid_display: false
```

Comment:
grid_display is false by default for HardDisk properties.  no need to set it here